src/Scripting/CSharpTest/ScriptTests.cs

```diff
@@ -687,6 +687,35 @@ public void TestCreateMethodDelegate()
             Assert.Equal(0, result);
         }
 
+        [Fact]
+        public async Task ScriptOptionsConfiguredForDebuggingResultInPdbEmitted()
+        {
+            try
+            {
+                var opts = ScriptOptions.Default.WithDebugInformation();
+                var script = await CSharpScript.RunAsync("throw new System.Exception();", opts);
+            }
+            catch (Exception ex)
+            {
+                // line information is only available when PDBs have been emitted
+                Assert.Contains("at Submission#0.<<Initialize>>d__0.MoveNext() in :line 1", ex.StackTrace);
+            }
+        }
+
+        [Fact]
+        public async Task ScriptOptionsNotConfiguredForDebuggingResultInPdbNotEmitted()
+        {
+            try
+            {
+                var script = await CSharpScript.RunAsync("throw new System.Exception();", ScriptOptions.Default);
+            }
+            catch (Exception ex)
+            {
+                // line information is only available when PDBs have been emitted
+                Assert.DoesNotContain("at Submission#0.<<Initialize>>d__0.MoveNext() in :line 1", ex.StackTrace);
```

Comment:
this was intended to be a negative test (does not contain the line number info). I will change it to a specific stack trace without line number, it will be much more explicit